content/influxdb/v2.0/reference/cli/influx/v1/auth/delete.md

```diff
@@ -0,0 +1,32 @@
+---
+title: influx v1 auth delete
+description: >
+  The `influx v1 auth delete` command deletes an authorization in the InfluxDB 1.x compatibility API.
+menu:
+  influxdb_2_0_ref:
+    name: influx v1 auth delete
+    parent: influx v1 auth
+weight: 101
+influxdb/v2.0/tags: [authorization]
+---
+
+The `influx v1 auth delete` command deletes an authorization in the [InfluxDB 1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+
+## Usage
+```
+influx v1 auth delete [flags]
+```
+
+## Flags
+| Flag |                   | Description                                                             | Input type | {{< cli/mapped >}}      |
+|:-----|:------------------|:------------------------------------------------------------------------|:----------:|:------------------------|
+| `-c` | `--active-config` | Config name to use for command                                          | string     | `$INFLUX_ACTIVE_CONFIG` |
```

Comment:
Does this delete the active config profile?

Reply:
No, this flag is a way to override which `influx` config to use with this specific command. The command will delete an authorization token.